cyw43/src/bluetooth.rs

```diff
@@ -0,0 +1,170 @@
+use embassy_time::{Duration, Timer};
```

Comment:
let me know if you want me to turn this file into a struct + impl or any other refactoring or if it is fine as is

Reply:
yeah, it'd be better to make them methods in `Runner` so you don't have to repeat the generics. (I wouldn't make a separate struct, Rust allows impl blocks for structs defined in other files)